kubectl-cloudflow/cloudflowapplication/cloudflowapplication.go

```diff
@@ -142,6 +141,12 @@ type CloudflowApplicationSpec struct {
 	LibraryVersion string            `json:"library_version,omitempty"`
 }
 
+// Version of the application descriptor format
+const Version = "1"
```

Comment:
Maybe add these versions (Version and LibraryVersion) in version.go? (Version as ApplicationDescriptorVersion) a lot of other version stuff is there, probably easy to find replace in build.

Reply:
+1 ..